client/js/src/main/scala/zio/zmx/client/frontend/views/DiagramView.scala

```diff
@@ -31,6 +31,44 @@ object DiagramView {
 
     private val titleVar = Var("")
 
+    def RemoveDiagram(d: DiagramConfig): HtmlElement =
+      a(
+        cls := "text-white font-bold text-xl py-2 px-4 mx-2 rounded text-center place-self-center",
+        "Remove Diagram",
+        onClick.map(_ => Command.RemoveDiagram(d)) --> Command.observer
+      )
+
+    def ChartConfig(d: DiagramConfig): HtmlElement =
+      div(
+        cls := "w-1/5 p-3 ml-2",
```

Comment:
I wonder if breaking the style string into a tuple for better style management:
Take this line and the tuple version would be `cls := ("w-1/5", "p-3", "ml-2")`

Reply:
@ithinkicancode I agree with trying to make the styling easier and more consistent. I have created #330 to see if the work in laminext styling would help us. Ideally I would like none of our UI views contain concrete styles, but only references to settings within a theme.